modules/ROOT/pages/platform/api/authentication.adoc

```diff
@@ -0,0 +1,121 @@
+[[aura-api-authentication]]
+= Authentication
+:description: This page describes how to authenticate requests to the Aura API.
+
+label:AuraDB-Enterprise[]
+label:AuraDS-Enterprise[]
+
+[TIP]
+====
+The Aura API is currently in Public Beta for Aura Enterprise customers only. If you wish to participate in the Public Beta, please contact your Customer Success Manager.
+====
+
+The Aura API uses OAuth 2.0 for API authentication.
+
+== Creating credentials
+
+. Navigate to the https://console.neo4j.io/[Neo4j Aura Console Account Details page] in your browser.
+. Select the *Create* button in the *OAuth 2.0 Clients* section.
+. Enter a *Client name*, and select *Create*.
+. Securely save the *Client ID* and *Client Secret* you are given in the resulting modal; you will need these for the next step.
+
+[CAUTION]
+====
+You cannot retrieve your secret after you close the modal, so save it securely.
+====
+
+== Authenticating API requests
+
+To authenticate API requests to the link:{neo4j-docs-base-uri}/aura/api/specification/[Aura API], you must provide a Bearer Token in the `Authorization` header of each request as shown below:
+
+`Authorization: Bearer <access_token>`
+
+[TIP]
+====
+You can use the *Authorize* button on the link:{neo4j-docs-base-uri}/aura/api/specification/[Aura API] page to authenticate and test endpoints directly using your client ID and client secret. The UI sets the `Authorization` header for you.
+====
+
+=== Token endpoint
+
+You can use the following `/oauth/token` endpoint to obtain a Bearer Token for authenticating API requests.
+
+Authentication to the token endpoint uses HTTP Basic Auth with your client ID as the username and client secret as the password.
+
+[CAUTION]
+====
+Both the request and response contain sensitive information and must be kept secure. 
+
+You are responsible for keeping the client credentials and access tokens confidential, whether in transit (by specifying HTTPS), if stored at rest, in log files, etc.
+====
+
+==== Request parameters
+
+* Method: `POST`
+* Token URL: `https://api.neo4j.io/oauth/token`
+
+==== Request header
```

Comment:
This should include details that you need the Authorization header `Authorization: Basic <client_id:client_secret base64 encoded>. https://stackoverflow.com/questions/35442816/what-is-u-flag-on-curl-actually-doing . We can then say that curl --user is doing that for you in the example below.

Reply:
@sampsonbryce myself and @nvitucci initially had mentioned the base 64 encoding of client_id:client_secret but decided that someone advanced enough to be wanting to do that manually would already know about it and therefore it was a little redundant and potentially confusing for newer users. 

Do you disagree with this strongly enough for us to reconsider adding it? Do you think it is most important mentioning it around the cURL example below as opposed to the **Authenticating API requests** above?

Reply:
actually you know what i think you're right, let me work on this